autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -1,22 +1,24 @@
 import * as React from 'react';
-import { useDispatch } from 'react-redux';
-import { RouteComponentProps } from '@reach/router';
+import { useDispatch, useSelector } from 'react-redux';
+import { RouteComponentProps, navigate } from '@reach/router';
 import Schedule from './Schedule/Schedule';
 import * as styles from './SchedulingPage.css';
 import ConfigureCard from './ConfigureCard/ConfigureCard';
 import SchedulePreview from './SchedulePreview/SchedulePreview';
 import CourseSelectColumn from './CourseSelectColumn/CourseSelectColumn';
 import setTerm from '../../redux/actions/term';
+import { RootState } from '../../redux/reducer';
```

Comment:
Can you fix these two linting errors? Afterwards we'll be good to go

Reply:
Fixed. Thanks for finding that for me!